### DISCUSSION TITLE
Change absolute paths resolution

### DISCUSSION THREAD

**Discussion by cokolele** [parcel-bundler/parcel#5896] [Q&A] [answered]
I'm updating parcel from the latest v1 to the nightly release and the problem I ran into is path resolution in the whole project.
The project is built exactly as the one described [here](https://v2.parceljs.org/features/module-resolution/#flow-with-absolute-or-tilde-resolution), except I didn't use Flow.

Changing path from "/..." to "/src/..." seems to solve the problem, but i would have to rewrite it everywhere.
Is there any way to change absolute path resolution from project root to (entry) root directory without installing flow ?
Just creating .flowconfig or adding alias didn't change anything.
I am sorry if this is trivial question or already answered, I can't find anything useful.

My entry point is HTML with ```<script type="module" src="/index.js"></script>```

![image](https://user-images.githubusercontent.com/26676195/108594793-4a065500-737c-11eb-884d-fcb4e7f9c50f.png)
![image](https://user-images.githubusercontent.com/26676195/108594645-7ff70980-737b-11eb-9abb-85e75be04e20.png)

**Comment by mischnic** [comment, marked as answer]
One possibility would be https://github.com/mischnic/parcel-resolver-root

**Reply by cokolele** [reply]
Great, thank you!

I hacked it by creating empty package-lock.json in /src directory so parcel thinks it's project root. Right now it seems to work without issues, are there any side effects to this ?
